Configuration/TCA/tx_styleguide_type.php

```diff
@@ -3,7 +3,7 @@
 return [
     'ctrl' => [
         'title' => 'Form engine - type',
-        'label' => 'uid',
+        'label' => 'title',
```

Comment:
I'd like to keep uid as label for consistency here: All other tables do that, too.